docs/source/tutorials/token_classification.ipynb

```diff
@@ -0,0 +1,730 @@
+    "from cleanlab.token_classification.rank import get_label_quality_scores, issues_from_scores \n",
+    "from cleanlab.internal.token_classification_utils import * \n",
+    "from cleanlab.token_classification.summary import * "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ad75b45",
+   "metadata": {},
+   "source": [
+    "## 2. Get `pred_probs` and `labels` \n",
+    "\n",
+    "`pred_probs` are out-of-sample model-predicted probabilities of the CoNLL-2003 dataset (including training, development, and testing dataset), obtained via cross-validation. To detect potential labels issues, we first get `pred_probs` and `labels`, which are both in nested-list format, such that: \n",
+    "\n",
+    "- `pred_probs` is a list of `np.arrays`, such that `pred_probs[i]` is the model-predicted probabilities for the tokens in the i'th sentence, and has shape `(N_i, K)`, where `N_i` is the number of word-level tokens of the `i`'th sentence. Each row of the matrix corresponds to a token `t` and contains the model-predicted probabilities that `t` belongs to each possible class, for each of the K classes. The columns must be ordered such that the probabilities correspond to class 0, 1, ..., K-1. \n",
+    "        \n",
+    "- `labels` is a list of lists, such that `labels[i]` is a list of given token labels of the `i`'th sentence. For dataset with K classes, labels must be in 0, 1, ..., K-1. All the classes (0, 1, ..., and K-1) MUST be present in ``labels[i]`` for some ``i``. \n",
+    "\n",
+    "Here, indicies are a tuple `(i, j)` unless otherwise specified, which refers to the `j`'th word-level token of the `i`'th sentence. Given that each sentence has different number of tokens, we store `pred_probs` and `labels` as `.npz` files, which can be easily converted to dictionaries. Use `read_npz` to retrieve `pred_probs` and `labels` in nested-list format. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4f4ebdc0",
+   "metadata": {},
+   "source": [
+    "<details><summary>Below is the code used to read the `.npz` file. </summary> \n",
+    "    \n",
+    "    def read_npz(filepath): \n",
+    "        data = dict(np.load(filepath)) \n",
+    "        data = [data[str(i)] for i in range(len(data))] \n",
+    "        return data </details> "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "ab9d59a0",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "def read_npz(filepath): \n",
+    "    data = dict(np.load(filepath)) \n",
+    "    data = [data[str(i)] for i in range(len(data))] \n",
+    "    return data "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "519cb80c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pred_probs = read_npz('pred_probs.npz') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1dc3150f",
+   "metadata": {},
+   "source": [
+    "## 3. Use cleanlab to find potential label issues \n",
```

Comment:
It's too early to start this section. The code is still processing the data to get labels. 

The "use cleanlab" section should be super concise and should only begin once the pred_probs, labels, tokens are all already properly-formatted objects (no more data processing code in that section)